app/services/api/v3/ecf/participants_query.rb

```diff
@@ -0,0 +1,99 @@
+                   .where(
+                     schedule: { cohort_id: with_cohorts.map(&:id) },
+                     induction_programme: {
+                       partnerships: {
+                         lead_provider_id: lead_provider.id,
+                         challenged_at: nil,
+                         challenge_reason: nil,
+                       },
+                     },
+                   )
+
+          scope = User
+                    .includes(:participant_identities, :teacher_profile, participant_profiles: %i[participant_profile_states schedule teacher_profile ecf_participant_eligibility ecf_participant_validation_data])
+                    .eager_load(participant_profiles: :induction_records)
+                    .joins(left_outer_join_participant_profiles)
+                    .joins(left_outer_join_participant_identities)
+                    .joins("JOIN (#{join.to_sql}) AS latest_induction_records ON latest_induction_records.latest_id = induction_records.id")
+                    .joins(left_outer_join_participant_profile_states)
+                    .distinct
+
+          scope = updated_since.present? ? scope.where(users: { updated_at: updated_since.. }) : scope
+          params[:sort].blank? ? scope.order("users.updated_at ASC") : scope
+        end
+
+        def participant
+          participants.find(params[:id])
+        end
+
+      private
+
+        attr_accessor :lead_provider, :params
+
+        def filter
+          params[:filter] ||= {}
+        end
+
+        def with_cohorts
+          return Cohort.where(start_year: filter[:cohort].split(",")) if filter[:cohort].present?
+
+          Cohort.where("start_year > 2020")
+        end
+
+        def updated_since
+          return if filter[:updated_since].blank?
+
+          Time.iso8601(filter[:updated_since])
+        rescue ArgumentError
+          begin
+            Time.iso8601(URI.decode_www_form_component(filter[:updated_since]))
+          rescue ArgumentError
+            raise Api::Errors::InvalidDatetimeError, I18n.t(:invalid_updated_since_filter)
+          end
+        end
+
+        def latest_induction_record_order
+          <<~SQL
+            PARTITION BY induction_records.participant_profile_id ORDER BY
+              CASE
+                WHEN induction_records.end_date IS NULL
+                  THEN 1
+                ELSE 2
+              END,
+              induction_records.start_date DESC,
+              induction_records.created_at DESC
+          SQL
+        end
+
+        def left_outer_join_participant_profiles
+          "LEFT OUTER JOIN participant_profiles ON participant_profiles.id = induction_records.participant_profile_id"
+        end
+
+        def left_outer_join_participant_identities
+          "LEFT OUTER JOIN participant_identities ON participant_identities.id = participant_profiles.participant_identity_id"
+        end
+
+        def left_outer_join_participant_profile_states
```

Comment:
chatted to @leandroalemao: we need to select all profile states regardless of latest